test/test_aw.py

```diff
@@ -0,0 +1,96 @@
+import FIAT
+import finat
+import numpy as np
+from gem.interpreter import evaluate
+from fiat_mapping import MyMapping
+
+
+def test_awnc():
+    ref_cell = FIAT.ufc_simplex(2)
+    ref_el_finat = finat.ArnoldWintherNC(ref_cell, 2)
+    ref_element = ref_el_finat._element
+    ref_pts = ref_cell.make_points(2, 0, 3)
+    ref_vals = ref_element.tabulate(0, ref_pts)[0, 0]
+
+    phys_cell = FIAT.ufc_simplex(2)
+    phys_cell.vertices = ((0.0, 0.0), (2.0, 0.1), (0.0, 1.0))
+    phys_element = ref_element.__class__(phys_cell, 2)
+
+    phys_pts = phys_cell.make_points(2, 0, 3)
+    phys_vals = phys_element.tabulate(0, phys_pts)[0, 0]
+
+    # Piola map the reference elements
+    J, b = FIAT.reference_element.make_affine_mapping(ref_cell.vertices,
+                                                      phys_cell.vertices)
+    detJ = np.linalg.det(J)
+
+    ref_vals_piola = np.zeros(ref_vals.shape)
+    for i in range(ref_vals.shape[0]):
+        for k in range(ref_vals.shape[3]):
+            ref_vals_piola[i, :, :, k] = \
+                J @ ref_vals[i, :, :, k] @ J.T / detJ**2
+
+    # Zany map the results
+    mppng = MyMapping(ref_cell, phys_cell)
+    Mgem = ref_el_finat.basis_transformation(mppng)
```

Comment:
Probably `mapping` is preferable to `mppng`.

Reply:
changed.